app/views/projects/index.html.erb

```diff
@@ -9,9 +9,10 @@
 <div class="center">
 <h2>Projects</h2>
   <%= search_form_for @search do |f| %>
+      <%= f.label :badge_status %>
+      <%= f.collection_select(:badge_status_matches, Project::BADGE_STATUS_CHOICE, :id, :name) %>
```

Comment:
I tried this change, and ransack fails to generate the `projects.badge_status` condition in the queries. Almost as bad, the result page fails to render the user's drop-down selection correctly.